cmake/Platform/Core/Libraries/ArduinoLibraryFactory.cmake

```diff
@@ -48,11 +48,11 @@ function(make_arduino_library VAR_NAME BOARD_ID LIB_PATH COMPILE_FLAGS LINK_FLAG
             endforeach ()
 
             if (LIB_INCLUDES)
-                string(REPLACE ";" " " LIB_INCLUDES "${LIB_INCLUDES}")
+                string(REPLACE ";" " " LIB_INCLUDES_SPACE_SEPARATED "${LIB_INCLUDES}")
```

Comment:
Only the compiler command line generated with `set_target_properties` below requires a space separated version of `LIB_INCLUDES`.
If  we replace the `;` in the cmake list `LIB_INCLUDES`, the `list(REMOVE_DUPLICATES LIB_INCLUDES)` call at the end of the function does not remove anything.